docs/reference/errors-and-warnings/NU5004.md

```diff
@@ -16,7 +16,7 @@ f1_keywords:
 
 ### Issue
 
-NuGet pack operation was performed using `-Symbol` option, but the folder containing the nuspec file contained no assembly files. 
+The NuGet pack operation was performed using `-Symbol` option. But the folder containing the nuspec file contained no assembly files. 
```

Comment:
Why did you break this up into two sentences? I think it read better as one sentence.